specs/latest/1.0/index.html

```diff
@@ -2204,6 +2204,10 @@ <h4>Buffer objects</h4>
         <dt class="idl-code">void bindBuffer(GLenum target, WebGLBuffer? buffer)
             <span class="gl-spec">(<a href="http://www.khronos.org/registry/gles/specs/2.0/es_full_spec_2.0.25.pdf#nameddest=section-2.9">OpenGL ES 2.0 &sect;2.9</a>, <a class="nonnormative" href="http://www.khronos.org/opengles/sdk/2.0/docs/man/xhtml/glBindBuffer.xml">man page</a>)</span>
         <dd>
+            If <code>buffer</code> was generated by a different <code>WebGLRenderingContext</code>
+            than this one, generates an <code>INVALID_OPERATION</code> error, and the current
+            binding remains untouched. <br><br>
```

Comment:
You can drop the "and the current binding remains untouched", since that's implied from the nature of all non-OOM error generation.

Reply:
OK. The recently-added spec text for attempting to bind deleted objects includes this, but I'll remove it from this pull request.